Firestore/Source/Public/FirebaseFirestore/FIRFieldValue.h

```diff
@@ -90,6 +91,14 @@ NS_SWIFT_NAME(FieldValue)
  */
 + (instancetype)fieldValueForIntegerIncrement:(int64_t)l NS_SWIFT_NAME(increment(_:));
 
+/**
+ * Creates a new `VectorValue` constructed with a copy of the given array of NSNumbers.
+ *
+ * @param values Create a `VectorValue` instance with a copy of this array of NSNumbers.
+ * @return A new `VectorValue` constructed with a copy of the given array of NSNumbers.
+ */
++ (FIRVectorValue *)vectorFromNSNumbers:(NSArray<NSNumber *> *)values NS_REFINED_FOR_SWIFT;
```

Comment:
Based on proposed feedback in other file, this would probably turn into:
```suggestion
+ (FIRVectorValue *)vectorFromArray:(NSArray<NSNumber *> *)array NS_REFINED_FOR_SWIFT;
```

Reply:
Implemented

Reply:
Could the remote branch be out of date? I'm still seeing `vectorFromNSNumbers` here instead of `vectorFromArray`.

Reply:
Updating–– I think `vectorWithArray`. may be more idiomatic and consistent with some of the factory methods in Apple's Foundation framework.

Reply:
I've corrected this name in all places to `vectorWithArray`

Reply:
Taking a look now. Thanks!